hil-test/tests/aes.rs

```diff
@@ -0,0 +1,74 @@
+//! AES Test
+
+#![no_std]
+#![no_main]
+
+use hil_test::esp_hal::{
+    aes::{Aes, Mode},
+    peripherals::Peripherals,
+};
+
+struct Context<'a> {
+    aes: Aes<'a>,
+}
+
+impl Context<'_> {
+    pub fn init() -> Self {
+        let peripherals = Peripherals::take();
+        let aes = Aes::new(peripherals.AES);
+
+        Context { aes }
+    }
+}
+
+#[embedded_test::tests]
```

Comment:
Maybe we can add a cfg here to cfg the tests away? Not sure if it will work though.

Reply:
Tried that earlier, but it fails.

Reply:
Damn :sob: